cranelift/codegen/src/isa/riscv64/lower.isle

```diff
@@ -624,12 +624,12 @@
 ;;;;;  Rules for `select`;;;;;;;;;
 (rule
   (lower (has_type ty (select c @ (value_type cty) x y)))
-  (gen_select ty (truthy_to_reg cty (normalize_cmp_value cty c)) x y))
+  (gen_select ty (truthy_to_reg cty (normalize_cmp_value cty c (ExtendOp.Zero))) x y))
 
 (rule 1
   (lower (has_type (fits_in_64 ty) (select (icmp cc a b @ (value_type in_ty)) x y)))
-  (let ((a Reg (normalize_cmp_value in_ty a))
-        (b Reg (normalize_cmp_value in_ty b)))
+  (let ((a Reg (normalize_cmp_value in_ty a cc))
+        (b Reg (normalize_cmp_value in_ty b cc)))
```

Comment:
Since only one rule needs this conversion, I would prefer to do the conversion explicitly and remove the `(convert)` directive.
```suggestion
  (let ((a Reg (normalize_cmp_value in_ty a (intcc_to_extend_op cc)))
        (b Reg (normalize_cmp_value in_ty b (intcc_to_extend_op cc))))
```